config/config.exs

```diff
@@ -6,7 +6,9 @@
 use Mix.Config
 
 # General application configuration
-config :cadet, ecto_repos: [Cadet.Repo]
+config :cadet,
+  ecto_repos: [Cadet.Repo],
+  updater: [interval: 5 * 60 * 1000]
```

Comment:
Can't easily use Timex in the config, since config is evaluated before dependencies are compiled. I don't think it's worth the trouble to implement a work around just to be able to use this pipe. Added a comment indicating that the value for `interval` is in milliseconds in fd9de8e.